login/service.go

```diff
@@ -262,7 +262,7 @@ func (keycloak *KeycloakOAuthProvider) ExchangeRefreshToken(ctx context.Context,
 // encodes the token and returns final URL to which we are supposed to redirect
 func (keycloak *KeycloakOAuthProvider) CreateOrUpdateIdentityAndUser(ctx context.Context, referrerURL *url.URL, keycloakToken *oauth2.Token, request *goa.RequestData, idpProvider oauth.IdentityProvider, config Configuration) (*string, *oauth2.Token, error) {
```

Comment:
should you also rename this method to `UpdateIdentityAndUser`, then?

Reply:
renaming might make @sbryzak 's refactor harder :)

Reply:
fair point